tests/memealyzer.spec.ts

```diff
@@ -0,0 +1,62 @@
+import { test, expect, WebSocket } from "@playwright/test";
+
+var uri: string = "http://localhost:1080/";
+
+interface Image {
+  id: string;
+  Id: string;
+}
```

Comment:
I really don’t like that two properties have almost the same property names. Is this avoidable?

Reply:
The image json is "id" and the event returned from SignalR is "Id". So I need them both for now. Will investigate why later.